scripts/asciidoc2html.py

```diff
@@ -181,7 +182,7 @@ def _build_website_file(self, root: pathlib.Path, filename: str) -> None:
 
     def _build_website(self) -> None:
         """Prepare and build the website."""
-        theme_file = (pathlib.Path('www') / 'qute.css').resolve()
+        theme_file = (pathlib.Path(REPO_ROOT, 'www') / 'qute.css').resolve()
```

Comment:
```suggestion
        theme_file = REPO_ROOT / 'www' / 'qute.css'
```

Reply:
This line was directly 'translated' (😅) from `os.path` to pathlib.

It used to be `theme_file = os.path.abspath(os.path.join('www', 'qute.css'))`.

Wasn't sure if the `resolve()` could be removed so just wanted to point that out. :-)

Reply:
It might have been needed, but now that we have `REPO_ROOT` as an absolute path, I don't think it is anymore.